### PR TITLE
Add additional instructions for creating git repo in existing project

### PR DESCRIPTION
**Pull request by sciencewhiz**

### CODE REVIEW

File: source/docs/software/basic-programming/git-getting-started.rst [hunk 1]
```diff
@@ -93,7 +93,7 @@ Now you'll want to open a PowerShell window and navigate to your project directo
 
 In the below example, we created a file called ``README.md`` with the contents of ``# Example Repo``. More details on the various commands can be found in the subsequent sections.
 
-.. note:: Replace the filepath ("C:\Users\ExampleUser9007\Documents\Example Folder") with the one you want to create the repo in, and replace the remote URL (https://github.com/ExampleUser9007/ExampleRepo.git) with the URL for the repo you created in the previous steps.
+.. note:: Replace the filepath ``"C:\Users\ExampleUser9007\Documents\Example Folder"`` with the one you want to create the repo in, and replace the remote URL (https://github.com/ExampleUser9007/ExampleRepo.git) with the URL for the repo you created in the previous steps.
```
Comment (Daltz333) [inline source/docs/software/basic-programming/git-getting-started.rst:96]:
```suggestion
.. note:: Replace the filepath ``"C:\Users\ExampleUser9007\Documents\Example Folder"`` with the one you want to create the repo in, and replace the remote URL (``https://github.com/ExampleUser9007/ExampleRepo.git``) with the URL for the repo you created in the previous steps.
```

**Review by Daltz333 — CHANGES_REQUESTED** [review]

File: source/docs/software/basic-programming/git-getting-started.rst [hunk 2]
```diff
@@ -109,7 +109,7 @@ In the below example, we created a file called ``README.md`` with the contents o
     > git remote add origin https://github.com/ExampleUser9007/ExampleRepo.git
     > git push -u origin master
 
-For FRC Robot projects, the above commands should be run in the root of a project :ref:`created by the VS Code WPILib Project Creator <docs/zero-to-robot/step-3/creating-benchtop-test-program-cpp-java:Creating a New WPILib Project>`.
+For FRC Robot projects, the above commands should be run in the root of a project :ref:`created by the VS Code WPILib Project Creator <docs/zero-to-robot/step-3/creating-benchtop-test-program-cpp-java:Creating a New WPILib Project>`. Replace ``echo "# ExampleRepo" >> README.md`` and ``git add README.md`` with ``git add .`` to add all existing files in the project.
```
Comment (Daltz333) [inline source/docs/software/basic-programming/git-getting-started.rst:112]: I feel like this language could be could be more clear to convey that the lines you are running are for a pre-existing project.

**Comment by Daltz333** [comment]
I was thinking something along the lines of a 

```
.. note:: In cases where you want to convert an existing project/directory to a git repo. Replace the following commands with these
```

File: source/docs/software/basic-programming/git-getting-started.rst (hunk 2 above)
Comment (sciencewhiz) [reply source/docs/software/basic-programming/git-getting-started.rst]: I made it even more obvious by putting instructions for both cases in tabs.

**Review by Daltz333 — APPROVED** [review]

**Pull request merged**